x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -135,7 +168,7 @@ const transformControlColumns = ({
           width={width ?? MIN_ACTION_COLUMN_WIDTH}
         />
       ),
-      width: actionColumnsWidth,
+      width: width ?? actionColumnsWidth,
```

Comment:
I changes this line to make to be able to define the width  on `const checkBoxControlColumn: ControlColumnProps = {...`